cmd/tg_bot/internal/handlers/text.go

```diff
@@ -0,0 +1,181 @@
+package handlers
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	tele "gopkg.in/telebot.v3"
+	"nodemon/cmd/tg_bot/internal"
+	"nodemon/pkg/messaging/pair"
+)
+
+func AddNewNodeHandler(
+	c tele.Context,
+	environment *internal.TelegramBotEnvironment,
+	requestType chan pair.RequestPair,
+	responsePairType chan pair.ResponsePair) error {
+
+	url := strings.TrimPrefix(c.Text(), "Add ")
+	if !strings.HasPrefix(url, "http") && !strings.HasPrefix(url, "https") {
+		return c.Send(
+			"Sorry, the url seems to be incorrect",
+			&tele.SendOptions{ParseMode: tele.ModeDefault},
+		)
+	}
```

Comment:
Use url.Parse for URL checking.

Reply:
Nope, excuse me. Use [url.ParseRequestURI](https://golang.org/pkg/net/url/#ParseRequestURI) for that.

Reply:
Please see the comment above